src/Middleware/HealthChecks.EntityFrameworkCore/ref/Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore.csproj

```diff
@@ -1,10 +1,10 @@
 <!-- This file is automatically generated. -->
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1</TargetFrameworks>
```

Comment:
We should note this either as a new breaking change announcement or as part of the EFCore announcement for dropping .NET Standard 2.0. This change makes total sense (since why have a healthcheck if EF isn't supported ;)) but we need to make sure it's well announced.

cc @divega @glennc